cypress/support/commands.js

```diff
@@ -39,6 +39,24 @@ Cypress.Commands.add('login', () => {
   cy.contains('Welcome back');
 });
 
+Cypress.Commands.add('toggleAll', () => {
+  cy.login();
+  cy.visit('/settings');
+  // cy.get('input[name="isLocked"]').click();
+  // cy.get('input[name="name"]').click();
+  cy.get('#privacy-settings')
+    .find('.toggle-not-active')
+    .each(element => {
+      return new Cypress.Promise(resolve => {
+        element.click();
+        setTimeout(() => {
+          resolve();
+        }, 300);
+      });
+    });
+  cy.get('#honesty-policy').find('button').click().wait(300);
```

Comment:
What are we waiting for here?

Reply:
Honestly not too sure. If I recall, I needed this to have the tests pass more often.

Reply:
Yeah, I understand.  I've run into timing issues a bunch of times.

What I suggest is that you figure out what should happen next and just `cy.contains` whatever that is.